dask/array/tests/test_linalg.py

```diff
@@ -364,3 +364,55 @@ def test_inv():
         assert_eq(res, scipy.linalg.inv(A))
         assert_eq(dA.dot(res), np.eye(shape, dtype=float))
 
+
+def _get_symmat(size):
+    np.random.seed(1)
+    A = np.random.random_integers(1, 20, (size, size))
+    lA = np.tril(A)
+    return lA.dot(lA.T)
+
+
+def test_solve_sym_pos():
+    import scipy.linalg
+
+    for shape, chunk in [(20, 10), (30, 6)]:
```

Comment:
This would be a good place to use `pytest.mark.parametrize`. That way if a test fails the shape and chunk will be part of the test output - slightly easier to debug.